features/step_definitions/user_steps.rb

```diff
@@ -402,3 +406,7 @@
   link = user_path(user)
   page.find(:css, %Q{a[href="#{link}?tab=activity"]}).trigger('click')
 end
+
+Then(/^I should not exist as a user$/) do
+  expect(@user).to be_nil
```

Comment:
I think we should reload user from db here to avoid possibility that we are dealing with cached data